include/unk_0201E7D8.h

```diff
@@ -0,0 +1,61 @@
+#ifndef POKEDIAMOND_UNK_0201E7D8_H
+#define POKEDIAMOND_UNK_0201E7D8_H
+
+#include "fx.h"
+
+
+struct CameraHistory
+{
+    s32 count;
+    s32 readIdx;
+    s32 writeIdx;
+    s32 writeIdxInit;
+    BOOL overrideEnabled;
+    BOOL enableUpdateX;
+    BOOL enableUpdateY;
+    BOOL enableUpdateZ;
+    VecFx32 * vecs;
+};
+
+struct CameraAngle
+{

```

Comment:
*tentatively* I would say this is *possibly* originally intended to be similar to a quaternion, which in that case unk_06 would be w, or it could just be padding to int which seems more likely